Assets/00_Content/Scripts/Interactables/Weapons/Axe.cs

```diff
@@ -58,7 +59,12 @@ public class Axe : PickUp, IUseable {
 		private void OnTriggerEnter(Collider other) {
 			if (isAttacking) {
 				IHittable hittable = other.GetComponentInParent<IHittable>();
-				hittable?.Hit(this);
+
+				if(hittable != null) {
+					Instantiate(particleSystemHit, other.transform.position, Quaternion.identity);
```

Comment:
Instead of instantiating a new particle system each hit, why not have one which plays in world space which is played on each hit?

Reply:
👍